packages/jest-reporters/src/GithubActionsReporter.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {GITHUB_ACTIONS} from 'ci-info';
+import type {AggregatedResult, TestResult} from '@jest/test-result';
+import BaseReporter from './BaseReporter';
+import type {Context} from './types';
+
+const lineAndColumnInStackTrace = /^.*?:([0-9]+):([0-9]+).*$/;
+
+function replaceEntities(s: string): string {
+  const substitutions: Array<[RegExp, string]> = [
+    [/%/, '%25'],
```

Comment:
```suggestion
    [/%/g, '%25'],
```

I think the modifier was missed on this one.